data/sheragi/archaeology missions.txt

```diff
@@ -1170,7 +1170,11 @@ mission "Sheragi Archaeology: The Box 7b"
 			`	"I see. I'll have to warn you though, following the footsteps of a doomed species can only get you so far.`
 				goto end
 			label truth
-			`	"A Sheragi black box? That's an incredible find! Be careful about the information it contains, though. They are a doomed species. Don't you humans have a legend about a cursed box? What was it... Pandora's Box?`
+			`	You produce an image of the cube that you found on your holopad. The curator stares at it, and then slowly turns to you with pity in his eyes.`
+			`	"This box is red," he says gently. "I read a thesis once on the prevalence of color blindness in humans, but-"`
+			`	You interrupt the curator and explain that 'black box', in human terminology, refers to a device used to collect and preserve flight data in the event of a catastrophic failure.`
+			`	Recognition seems to dawn on him.`
```

Comment:
Should this still be on its own line, or should it be tacked onto the previous line?

Reply:
It seems a little short to be its own line but it is a separate idea.